.circleci/config.yml

```diff
@@ -378,6 +378,14 @@ jobs:
           at: ~/player
       - auto_shipit
 
+  bench_js:
+    executor: base
```

Comment:
For now, since the benchmarks are relatively small can/should we use a smaller executor?